doc/en/components/grids/_shared/excel-style-filtering.md

```diff
@@ -0,0 +1,1017 @@
+---
+title:  Excel Style Filtering in {Platform} {ComponentTitle}- {ProductName}
+_description: Learn how to configure Excel filtering in {Platform} {ComponentTitle}. You can enable/disable various options and customize the Excel style filter menu the way you want.
+_keywords: excel like filter, {Platform}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "PivotGrid", "HierarchicalGrid"]
+---
+
+# Excel Filtering in {Platform} {ComponentTitle}
+
+The grid Excel filtering provides an Excel like filtering UI for any Angular Material table like the {ComponentTitle}.
+
+## {Platform} {ComponentTitle} Excel Style Filtering Example
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:620px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-1"
+           github-src="{ComponentSample}/excel-style-filtering-sample-1" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-1"
+           github-src="{ComponentSample}/excel-style-filtering-sample-1" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-1"
+           github-src="{ComponentSample}/excel-style-filtering-sample-1" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.